### PR TITLE
chore: deprecate

### PR DESCRIPTION
**Pull request by danez**
Fixes: https://github.com/netlify/pod-compute/issues/55

- Remove renovate
- Remove label workflow
- Remove foss workflow
- Revert webpack5 upgrade from renovate (why did it do a major update anyway)
- Change [README.md](https://github.com/netlify/netlify-lambda/blob/89b04c41ecd5d7dadd74d144165e426586f3af54/README.md)  message about deprecation
- Add [MIGRATE.md](https://github.com/netlify/netlify-lambda/blob/89b04c41ecd5d7dadd74d144165e426586f3af54/MIGRATE.md)

After this I will be releasing one more patch version with all the security updates.

### CODE REVIEW

**Review by klavavej — CHANGES_REQUESTED** [review]
@danez Thanks for the review request! Nothing major but I left a few suggestions for smoothing out the flow and cleaning up the spelling and grammar so I'm requesting changes.

One thing I suggested in a couple places is removing the word "simply". This well-intentioned word can backfire and make people who find the directions confusing feel even worse about being confused. In general it's better to convey the simplicity of instructions by making the instructions themselves simple to follow rather than telling people explicitly that the process is simple.

**Comment by danez** [comment]
Thanks for the detailed review. I applied all the changes.

File: README.md [hunk 1]
```diff
@@ -1,560 +1,10 @@
...
-## Changelog
-
-- v1.0: https://twitter.com/Netlify/status/1050399820484087815 Webpack 4 and Babel 7
-- v1.1: https://twitter.com/swyx/status/1069544181259849729 Typescript support
-- v1.2: https://twitter.com/swyx/status/1083446733374337024 Identity emulation (& others)
-- v1.3: https://github.com/netlify/netlify-lambda/releases/tag/v1.3.0
-- v1.4: New timeout feature https://github.com/netlify/netlify-lambda/pull/116
-- v1.5: Catch raw requests - a very common error for first time users pinging `localhost:9000` instead of `localhost:9000/.netlify/functions/myfunction` https://github.com/netlify/netlify-lambda/commit/bfebc0921a45d4f730b910b680e40e04928f7c29#diff-3288939317efd62bfc509440d662cacaR191
-- v1.6: New `install` command https://mobile.twitter.com/swyx/status/1162038490298818562 
+*So long, and thanks for all the fish!*
```
[the remark this answers is not in the document]
Comment (klavavej) [reply README.md:7]: @danez - here's the one suggestion that wasn't committed. I'd still recommend removing this pop culture reference

**Review by klavavej — DISMISSED** [review]
@danez - I see [one suggestion that wasn't committed](https://github.com/netlify/netlify-lambda/pull/606#discussion_r1105109640) (the suggestion to remove the pop culture reference from the README). I assume this was an accident since your last comment said you applied all the suggested changes, but please let me know if you skipped that one on purpose. 

I'm re-upping that suggestion to remove the pop culture reference but otherwise this looks good to me - thanks for applying my suggestions ✨

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (danez) [reply README.md]: Sorry I missed that.

**Review by klavavej — APPROVED** [review]
Thanks for that last change @danez !

**Pull request merged**